### PR TITLE
Ninja added

### PR DESCRIPTION
**Pull request by RikNieuwoudt**
[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/2312)

### CODE REVIEW

**Comment by RikNieuwoudt** [comment]
@casey-collab

File: english/ninja/ninja.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[../buy-gameboy/buy-gameboy.md](gameboy)
```
Comment (casey-collab) [inline english/ninja/ninja.md]: It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.

Comment (casey-collab) [inline english/ninja/ninja.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Pull request closed without merging**